src/services/vehicleService.ts

```
@@ -1,15 +1,14 @@
 const SERVICE = "vehicle"
 
-export async function getVehicle(guid: string): Promise<vehicleDetials | undefined> {
+export async function getVehicle(guid: string): Promise<vehicleDetails | undefined> {
   try {
     const rez = await axiosInstance.get(`/${SERVICE}/${guid}`)
     return rez.data
   }
   catch (error) {
-    alert(error)
-    return undefined
+    throw error
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)